### PR TITLE
Add sampling option to UIComponent

### PR DESCRIPTION
**Pull request by aunpyz**
# PR Details
UIComponent sampling option

## Description
I add the sampling option just like in SpriteComponent to UIComponent
Now we can select
- Point  (Nearest)
- Linear
- Anisotropic
image

Comparison between Point(left) and Linear sampler(right)
image

## Motivation and Context
I see the option is available only in SpriteComponent and at that time I want to have the same thing on UIComponent for my little 2D project
I'm quite new to Stride and maybe this option is available somewhere, but I can't find it so I just implement it.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/stride3d/stride?pullRequest=1045) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/stride3d/stride?pullRequest=1045) before we can accept your contribution.<br/><br/>:x: aunpyz [sign now](https://cla.dotnetfoundation.org/stride3d/stride?pullRequest=1045)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/stride3d/stride?pullRequest=1045) it.</sub>

**Comment by tebjan** [comment]
@xen2 I would wait before merging this PR, since the it doesn't look finished. See my comments in code...

**Comment by xen2** [comment]
Sorry, I merged by mistake, we'll need to recreate another PR

**Comment by aunpyz** [comment]
@tebjan what is the comment? I would love to see it.
Maybe it was removed from merging PR by mistake?

**Comment by tebjan** [comment]
They are inline comments, you should see them above this discussion or if you click on changes...

**Comment by aunpyz** [comment]
Looking into file changes and I still see no comment.
Is it either `<summary>` or `<userdoc>`?

**Comment by aunpyz** [comment]
Yes. It's where I look for your comments.
And sorry I can't find it still
> 
> 
> @xen2 I would wait before merging this PR, since it doesn't look finished. See my comments in code...

This is the topmost comment in this PR of you
FYI: I did check for comment to all my 3 commits and on changes in 'Files changed' tab

**Comment by tebjan** [comment]
oh wow, sorry @aunpyz and @xen2 i didn't see that i have to press "Submit Review"... i think i was the only one who saw the comments. Now check again please.

File: sources/engine/Stride.UI/Rendering/UI/UIRenderFeature.cs [hunk 1]
```diff
@@ -247,12 +247,23 @@ private void DrawInternal(RenderDrawContext context, RenderView renderView, Rend
                 }
                 context.CommandList.SetRenderTarget(renderingContext.DepthStencilBuffer, renderingContext.RenderTarget);
 
+                var samplerState = context.GraphicsDevice.SamplerStates.LinearClamp;
```
[the remark this answers is not in the document]
Comment (aunpyz) [reply sources/engine/Stride.UI/Rendering/UI/UIRenderFeature.cs:250]: Oh my bad, I added Anisotropic last minute to match the option of `SpriteComponent`'s `SpriteSampler`. About conponent processor I'll take a look at that ASAP, but initially I followed `SpriteRenderFeature` logic of setting up `SamplerState`

[the remark this answers is not in the document]
Comment (tebjan) [reply sources/engine/Stride.UI/Rendering/UI/UIRenderFeature.cs:250]: if it is the same as in SpriteRenderFeature, then it's totally fine.

**Pull request merged**